evm/script/ConfigureWormholeNtt.s.sol

```diff
@@ -0,0 +1,146 @@
+            );
+            require(config[i].inboundLimit != 0, "Invalid inbound limit");
+
+            // Make sure we don't configure the same chain twice.
+            require(!duplicateChainIds[config[i].chainId], "Duplicate chain ID");
+            duplicateChainIds[config[i].chainId] = true;
+
+            if (config[i].chainId == params.wormholeChainId) {
+                nttManager = INttManager(config[i].nttManager);
+                wormholeTransceiver = IWormholeTransceiver(config[i].wormholeTransceiver);
+            }
+        }
+    }
+
+    function configureWormholeTransceiver(
+        ChainConfig[] memory config,
+        ConfigParams memory params
+    ) internal {
+        for (uint256 i = 0; i < config.length; i++) {
+            ChainConfig memory targetConfig = config[i];
+            if (targetConfig.chainId == params.wormholeChainId) {
+                continue;
+            } else {
+                // Set relayer.
+                if (targetConfig.isWormholeRelayingEnabled) {
+                    wormholeTransceiver.setIsWormholeRelayingEnabled(targetConfig.chainId, true);
+                    console2.log("Wormhole relaying enabled for chain", targetConfig.chainId);
+                } else if (targetConfig.isSpecialRelayingEnabled) {
+                    wormholeTransceiver.setIsSpecialRelayingEnabled(targetConfig.chainId, true);
+                    console2.log("Special relaying enabled for chain", targetConfig.chainId);
+                }
+
+                // Set peer.
+                wormholeTransceiver.setWormholePeer(
+                    targetConfig.chainId, toUniversalAddress(targetConfig.wormholeTransceiver)
+                );
+                console2.log("Wormhole peer set for chain", targetConfig.chainId);
+
+                // Set EVM chain.
+                if (targetConfig.isEvmChain) {
+                    wormholeTransceiver.setIsWormholeEvmChain(targetConfig.chainId);
+                    console2.log("EVM chain set for chain", targetConfig.chainId);
+                } else {
+                   revert("Non-EVM chain is not supported yet");
+                }
+            }
+        }
+    }
+
+    function configureNttManager(ChainConfig[] memory config, ConfigParams memory params) internal {
+        for (uint256 i = 0; i < config.length; i++) {
+            ChainConfig memory targetConfig = config[i];
+            if (targetConfig.chainId == params.wormholeChainId) {
+                continue;
+            } else {
+                // Set peer.
+                nttManager.setPeer(
+                    targetConfig.chainId, toUniversalAddress(targetConfig.nttManager), targetConfig.decimals
+                );
+                console2.log("Peer set for chain", targetConfig.chainId);
+
+                // Configure the inbound limit.
+                nttManager.setInboundLimit(targetConfig.inboundLimit, targetConfig.chainId);
```

Comment:
So by default every other chain will have the same inbound limit coming from a given chain?

Reply:
This script is executed once for each chain, so the integrator could modify the inbound limit for each chain that it's registering.